pkg/arvo/app/clock.hoon

```diff
@@ -63,6 +75,13 @@
 ++  on-arvo
   |=  [=wire =sign-arvo]
   ^-  (quip card:agent:gall _this)
+  ?:  ?=(%wake +<.sign-arvo)
+    :_  this  :_  ~
+    ?^  error.sign-arvo
+      :_  this  :_  ~
```

Comment:
It looks like this duplicates the state and null-terminator in the product.

Reply:
indeed, mis-paste